client/components/plans/plan-actions/index.jsx

```diff
@@ -172,9 +173,17 @@ module.exports = React.createClass( {
 		return (
 			<div>
 				<button className="button is-primary plan-actions__upgrade-button"
-					onClick={ this.handleAddToCart.bind( null, this.cartItem( { isFreeTrial: false } ), 'button' ) }>
-						{ this.translate( 'Upgrade Now', { context: 'Store action' } ) }
+					onClick={ this.handleAddToCart.bind( null, this.cartItem( { isFreeTrial: true } ), 'button' ) }>
+						{ this.translate( 'Start Free Trial', { context: 'Store action' } ) }
 				</button>
+
+				<small className="plan-actions__trial-period">
+					{ this.translate( 'Try it free for 14 days, no credit card needed, or', { context: 'Store action' } ) }&nbsp;
+					<a href='#'
+						onClick={ this.handleAddToCart.bind( null, this.cartItem( { isFreeTrial: false } ), 'link' ) }>
+						{ this.translate( 'upgrade now', { context: 'Store action' } ) }
```

Comment:
What do you think about combining these into a single `this.translate` call? It might be easier to read and translate.

```js
this.translate( 'Try it free for 14 days, no credit card needed, or {{a}}upgrade now{{/a}}.', {
  context: 'Store action',
  components: {
    a: <a
          href="#"
          onClick={ this.handleAddToCart.bind( null, this.cartItem( { isFreeTrial: false } ), 'link' ) } />
  }
} )
```